addons/mail/static/src/models/message_view/message_view.js

```diff
@@ -17,6 +17,9 @@ registerModel({
             this.update({
                 isHighlighted: true,
                 highlightTimeout: this.env.browser.setTimeout(() => {
```

Comment:
Note that to be completely clean, we might want to clear `this.highlightTimeout` on willDelete of this model, so that the call does not even happen.